doc/examples/nonlinear_from_rest/submit_multiple_check.py

```diff
@@ -0,0 +1,41 @@
+from time import sleep
+
+from fluiddyn.clusters.legi import Calcul8 as Cluster
+
+aspect_ratios = [0.875, 1.0]
+nb_elements = 16
+order = 10
+end_time = 4000
+nb_procs = 10
+
+Ra_c_tests = {0.875: 2.855e8, 1.0: 1.875e8}
+
+cluster = Cluster()
+
+cluster.commands_setting_env = [
+    "PROJET_DIR=/fsnet/project/meige/2020/20CONVECTION",
+    "source /etc/profile",
+    "source $PROJET_DIR/Software/miniconda3/etc/profile.d/conda.sh",
+    "conda activate",
+    "export NEK_SOURCE_ROOT=$HOME/Documents/Nek5000",
+    "export PATH=$PATH:$NEK_SOURCE_ROOT/bin",
+    "export FLUIDSIM_PATH=$PROJET_DIR/numerical/",
+]
+
+
+for aspect_ratio in aspect_ratios:
+
+    Ra_c_test = Ra_c_tests[aspect_ratio]
```

Comment:
```suggestion
for aspect_ratio, Ra_c_test in zip(aspect_ratios, Ra_c_tests)
```

Your method is fine and works well, but I often tend to use I there `zip` or `itertools.product` (depending on how I want to parametrize).